drivers/rilmodem/sim.c

```diff
@@ -664,7 +664,10 @@ static void sim_status_cb(struct ril_msg *message, gpointer user_data)
 				__func__, status->card_state);
 
 	/* TODO(CDMA): need some kind of logic to set the correct app_index */
-	search_index = status->gsm_umts_index;
+	if (status->gsm_umts_index < 0)
```

Comment:
Error from compiler here:

drivers/rilmodem/sim.c:667:29: error: comparison of unsigned expression < 0 is always false [-Werror=type-limits]